src/api/comment/content-types/comment/lifecycles.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+
+    async afterCreate(event) {
+
+        const ideaCard = event.result.idea_card
+        const ideaName = event.result.idea_card.ideaName
+        const commentAuthor = event.result.author
+        const commentText = event.result.text
+        const timeCreated = event.result.createdAt
+
+        await strapi.entityService.create('api::notification.notification', {
+            data: {
+              Title: commentAuthor + " commented on " + ideaName,
+              Content: commentText,
+              Tag: "New Comment",
+              idea_card: ideaCard.id,
+              TimeCreated: timeCreated,
+              Read: false,
```

Comment:
how can we bind the notification to its appropriate user?
for example only idea_author and people interested in the idea would want to get notified but not everyone else

Reply:
I think the requirement from product is notification will only be sent to 1 user. In the case of comment, only the idea owner will be notified. In the case of applicant, only the owner of the opportunity will be notified. Too many notifications can cause fatigue and people will start ignoring them.

Reply:
> I think the requirement from product is notification will only be sent to 1 user. In the case of comment, only the idea owner will be notified. In the case of applicant, only the owner of the opportunity will be notified. Too many notifications can cause fatigue and people will start ignoring them.

good point but my question is about the viability of changing the current solution we want to include multiple recipients of the same notification? (for example in the chat section someone might tag two people in the same comment)